app/frontend/src/components/apiAccess/Step1.vue

```diff
@@ -1,21 +1,89 @@
 <template>
-  <v-container>
-    <h2>API Access for Common Services</h2>Contents to go here
-    <br />
-    <br />
-    <br />
-    <br />
-    <v-btn color="primary" @click="setStep(2)">Get Token for Dev</v-btn>
+  <v-container class="pl-0">
+    <h2 class="pb-8">API Access for Common Services</h2>
+    <p>Your application has a service client with access to:</p>
+    <ul>
+      <li>Common Hosted Email Service</li>
+      <li>Common Document Generation Service</li>
```

Comment:
May be worth alphabetizing this list, but that's a UX decision.

Reply:
Noting for discussion with Mina later